integration-tests/steps/stack_groups.py

```diff
@@ -266,6 +277,12 @@ def step_impl(context, first_stack, second_stack):
     assert creation_times[stacks[0]] < creation_times[stacks[1]]
 
 
+@then('stack_group drift status is "{desired_status0}" and "{desired_status1}"')
+def step_impl(context, desired_status0, desired_status1):
+    assert context.output[0]["StackDriftStatus"] == desired_status0
+    assert context.output[1]["StackDriftStatus"] == desired_status1
```

Comment:
This is pretty inflexible. I'd recommend doing something like this:
```suggestion
@then('stack_group drift statuses are each one of "statuses"')
def step_impl(context, statuses):
    status_list = [status.strip() for status in statuses.split(',')]
    for output in context.output:
        assert output["StackDriftStatus"] in status_list
```

Reply:
Alternatively, you could assign the whole dictionary returned from `drift_detect` to `context.output` and then you could just make the step `"stack_group drift status for stack "{stack_name}" is "{desired_status}"` and then just have two `then` clauses. That would be more precise, as you could grab the status by stack name. Honestly, that would probably be a better test, actually.